googlepubsub/json_schemas/message.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://asyncapi.com/bindings/amqp/channel.json",
```

Comment:
Change `$id` to `http://asyncapi.com/bindings/googlepubsub/message.json`

Reply:
Done.